CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 
 #### Plugins
 
+- Introduced the new **AI Proxy** plugin that enables simplified integration
+  with various AI provider Large Language Models.
+
```

Comment:
Please put changelog here: https://github.com/Kong/kong/tree/master/changelog/unreleased/kong/

Reply:
Done it, I understand now.